.github/workflows/build.yml

```diff
@@ -51,14 +51,12 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get -y install gcc-8 g++-8
-        python -m pip install --upgrade pip setuptools wheel
```

Comment:
FYI @dkeeney added the setuptools>=34.4.0 to requirements and removed the manual upgrade from CI (let's see if other pip,wheel break and need specific version too)